portfolio/src/main/webapp/script.js

```diff
@@ -41,26 +41,41 @@ function getComments() {
 
   fetch('/comments').then(response => response.json()).then((cmtContainer) => {
 
-    if (!cmtContainer) return;
+    const commentsArray = Array.from(cmtContainer);
+
+    console.log(commentsArray);
+
+    if (!(commentsArray && commentsArray.length)) return;
 
     const commentsOutput = document.getElementById("comments-output")
       .appendChild(document.createElement("h2")
         .appendChild(document.createTextNode("Comments")).parentElement);
 
     const cmtDOM = document.createElement("ul");
 
-    const commentsArray = Array.from(cmtContainer);
-
     commentsArray.forEach(cmt => {
       cmtDOM.appendChild(document.createElement("li").appendChild(
         document.createTextNode(cmt)).parentElement);
     });
 
     commentsOutput.appendChild(cmtDOM);
+    return false;
   });
 }
 
-window.onload = function() {
+window.addEventListener("DOMContentLoaded", function() {
+  
   randQuote();
   getComments();
-}
\ No newline at end of file
+
+  const cmt_form = document.getElementById("cmt_form");
+  const cmt_del_form = document.getElementById("cmt_del_form")
+
+  document.getElementById("cmt_button").addEventListener("click", function() {
```

Comment:
Glad to see you using the DOM API effectively! This can actually be done by giving your submit buttons `type="submit"` in the HTML though.